docs/concepts/resource-manager.md

```diff
@@ -249,11 +249,10 @@ In addition to the origin annotation, all objects managed by the resource manage
 
 #### Compression
 
-The number and size of manifests for a `ManagedResource` can accumulate to a considerable amount which leads to increased
-`Secret` data. A decent compression algorithm helps to reduce the footprint of such `Secret`s and the load they put on
-`etcd`, the `kube-apiserver`a and client caches.
-We found [Brotli](https://github.com/google/brotli) to be a suitable candidate for most use cases. When the `gardener-resource-manager`
-detects a data key with the known suffix `.bro`, it automatically un-compresses the data first before processing the contained manifest.  
+The number and size of manifests for a `ManagedResource` can accumulate to a considerable amount which leads to increased `Secret` data.
+A decent compression algorithm helps to reduce the footprint of such `Secret`s and the load they put on `etcd`, the `kube-apiserver`, and client caches.
+We found [Brotli](https://github.com/google/brotli) to be a suitable candidate for most use cases (see comparison table [here](https://github.com/gardener/gardener/pull/9868)). 
```

Comment:
I agree with your statement and understand your doubts. 

Concrete compression rates were collected because the usage or Brotli was questioned compared to Gzip. Compression speed was not recorded, as we relied on Brotli being acknowledged as an efficient and quick compression algorithm, see for instance [ref1](https://blog.cloudflare.com/results-experimenting-brotli) and [ref2](https://medium.com/@bansal.suneet/brotli-vs-gzip-compression-surprising-compression-result-brotli-power-782aac2ee29f).

If you don't have any particular concerns, I'd argue that these insights sufficiently attest our decision. WDYT?

Reply:
The choice of compression algorithm has quite some implications in terms of memory saved or CPU burnt. Therefore, I think it is a good idea to make a conscious decision about it. This decision and the trade-offs should be documented so that interested people can also figure them out later on.
That being said, my personal preference would be to define a simple scenario measure it and document it as part of this pull request, e.g. as you have already done in the pull request description. However, it is also valid to point to prior work, i.e. external resources, and argue that someone else has already done this work.
My main point is we should document why we chose what we chose. It is fine to state that we just focussed on the compression rate and relied on Brotli being efficient as described in your references, but please add this then to the description. Then, if someone wants to look into the CPU consumption piece it is clear that this is fairly open.

Reply:
Okay, thank. I added it to the PR description.